ephios/settings.py

```diff
@@ -16,44 +18,59 @@
 except ImportError:
     import importlib.metadata as importlib_metadata
 
+# BASE_DIR is the directory containing the ephios package
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
+
 env = environ.Env()
 # for syntax see https://django-environ.readthedocs.io/en/latest/
 # read env file from ENV_PATH or fall back to a .env file in the project root
 env_path = env.str("ENV_PATH", default=os.path.join(BASE_DIR, ".env"))
 print(f"Loading ephios environment from {Path(env_path).absolute()}")
 environ.Env.read_env(env_file=env_path)
 
-SECRET_KEY = env.str("SECRET_KEY")
 DEBUG = env.bool("DEBUG")
-ALLOWED_HOSTS = env.list("ALLOWED_HOSTS")
+if DEBUG:
+    DATA_DIR = env.str("DATA_DIR", os.path.join(BASE_DIR, "data"))
+else:
+    # DATA_DIR must be set explicitly in production
+    DATA_DIR = env.str("DATA_DIR")
+
+LOG_DIR = env.str("LOG_DIR", os.path.join(DATA_DIR, "logs"))
+MEDIA_ROOT = env.str("MEDIA_ROOT", os.path.join(DATA_DIR, "media"))
+STATIC_ROOT = env.str("STATIC_ROOT", os.path.join(DATA_DIR, "static"))
```

Comment:
I think we should keep to option to specify the static directory as people may want to put that somewhere under /var/www or similiar

Reply:
we are keeping it! You can still configure STATIC_ROOT!